make/common/templates/ui/env

```diff
@@ -7,3 +7,4 @@ ADMINSERVER_URL=$adminserver_url
 UAA_CA_ROOT=/etc/ui/certificates/uaa_ca.pem
 _REDIS_URL=$redis_url
 SYNC_REGISTRY=false
+CHART_CACHE_DRIVER=$chart_cache_driver
```

Comment:
How is it used?

Reply:
‘memory' or 'redis'. If it is redis, then try to read _REDIS_URL